lib/client.js

```diff
@@ -75,9 +81,19 @@ extend(Raven.prototype, {
     }
 
     registerExceptionHandler(this, cb);
-    if (opts && opts.unhandledRejection) {
-      registerRejectionHandler(this, cb);
+    if (opts) {
+      if (opts.unhandledRejection) {
+        registerRejectionHandler(this, cb);
+      }
+
+      if (opts.autoBreadcrumbs) {
```

Comment:
This should be able to work like this:

```javascript
Raven.config('your-dsn', {
  autoBreadcrumbs: true // everything
});
```

Where in a future version, `autoBreadcrumbs: true` becomes the default.